varats/varats/tools/research_tools/research_tool.py

```diff
@@ -490,6 +490,17 @@ def verify_install(self, install_location: Path) -> bool:
             True, if the tool was correctly installed
         """
 
+    def verify_build(
+        self, build_type: BuildType, build_folder_suffix: tp.Optional[str]
+    ) -> bool:
+        """
+        Verify if the research tool was built correctly for a given build_type.
+
```

Comment:
```suggestion

            build_type: which type of build should be used, e.g., debug,
                        development or release
            build_folder_suffix: a suffix that is appended to the build folder
```

PS: not sure if this is correctly aligned, gh makes it hard so see this